tests/test_modeling_hubert.py

```diff
@@ -579,3 +645,24 @@ def test_inference_ctc_batched(self):
             "sweat covered brion's body trickling into the tight loin cloth that was the only garment he wore",
         ]
         self.assertListEqual(predicted_trans, EXPECTED_TRANSCRIPTIONS)
+
+    def test_inference_classification_batched(self):
+        model = HubertForSequenceClassification.from_pretrained("anton-l/hubert-base-s3prl-superb-ks").to(torch_device)
+        processor = Wav2Vec2Processor.from_pretrained(
+            "anton-l/hubert-base-s3prl-superb-ks", do_lower_case=True, return_attention_mask=True
+        )
+        # TODO: use SUPERB Keyword Spotting instead
```

Comment:
yes that would be nice! Feel free to create a dummy testing dataset for this one :-)